zkevm-circuits/src/state_circuit/multiple_precision_integer.rs

```diff
@@ -67,6 +67,7 @@ impl Config<Address, N_LIMBS_ACCOUNT_ADDRESS> {
         value: Address,
     ) -> Result<(), Error> {
         for (i, &limb) in value.to_limbs().iter().enumerate() {
+            region.name_column(|| format!("STATE_MPI_limb[{}]_address", i), self.limbs[i]);
```

Comment:
Same comment to follow approach in https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1167/files

Reply:
fixed in [1b5aa41](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1152/commits/1b5aa4193380414a8378031399be7a6eddfac11a)